packages/desktop/src/app.js

```diff
@@ -98,6 +98,17 @@ const App = () => {
     [authorizedFetch]
   );
 
+  const markChannelRead = React.useCallback(
```

Comment:
Defining "actions" in the top scope for now. You could just as well pass down `serverConnection` and call `serverConnection.send("foo")` directly in the UI code.